src/theme/DocCard/index.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import clsx from "clsx";
+import Link from "@docusaurus/Link";
+import {
+  findFirstCategoryLink,
+  useDocById,
+} from "@docusaurus/theme-common/internal";
+import isInternalUrl from "@docusaurus/isInternalUrl";
+import { translate } from "@docusaurus/Translate";
+import styles from "./styles.module.css";
+function CardContainer({ href, children, border }) {
+  clsx("card padding--lg", styles.cardContainer);
+  var className = clsx("card padding--lg", styles.cardContainer);
+  if (border == "alt") {
```

Comment:
```suggestion
  if (border == "highlight") {
```

non-blocking: What do you think about using the value `highlight` instead of `alt` to signify which items should be....highlighted? 😎 

(Note that if we make this change, we also need to make it in the item descriptions)

Reply:
Agreed here. Resolving in latest commit 👍